### ISSUE TITLE
Access to response object + headers

### ISSUE THREAD

**Issue by mailbackwards** [charlesthk/python-mailchimp#88]
Some MailChimp data that used to live in the response body now lives in the response headers, for instance [the `web_id` of a campaign](http://stackoverflow.com/questions/39324653/no-web-id-in-mailchimp-api-v3). I need this data but it does not appear that it's available using this package. Is it or will it be possible to expose the response object or at least the response headers in question?

**Comment by stephenross** [comment]
I am currently looking into what methods the dashboard for a campaign can be accessed but having trouble with my account. Once I hear back from MailChimp about that, I will try to get to the dashboard in some other way, though it may also be a good idea to submit a support request to get that information exposed or to allow the ID that you receive from the api to function as well. As a quick test while I wait to hear back from support, have you tried replacing the decimal number in that link with the `id` provided by the API?

**Comment by mailbackwards** [comment]
Yes, I tried replacing the link's web ID with the API `id`, and unfortunately that didn't work. Thank you for the feedback, I'll submit a support request as well.

**Comment by bochecha** [comment]
Another useful thing to have is the link to a list member in the dashboard, which also is only available in the response headers:

```python
member_url = 'https://us8.api.mailchimp.com/3.0/lists/{list}/members/{member}/'.format(list='<list id>, member='<subscriber has>')

member = requests.get(member_url, auth=('<user>', '<api key>')
dashboard_url = member.links['dashboard']['url']
```
